subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -605,6 +648,11 @@ static void isr_rx(void *param)
 			}
 			payload_tail = payload_index;
 		}
+
+		str_idx++;
+		if (str_idx < lll->stream_count) {
+			lll->stream_curr = str_idx;
```

Comment:
As `lll->stream_curr` has global scope, to avoid out-of-bound value (i.e. > `lll->stream_count`) being assigned during runtime the value to be set has to be compared before assignment. Hence, a local variable used to increment, check and assign to `lll->stream_curr`.